lib/arweave/clientUploadToArweave.ts

```
@@ -0,0 +1,28 @@
+
+const clientUploadToArweave = async (file: File): Promise<string | null> => {
```

ESLint

[error] 'file' is defined but never used.

(@typescript-eslint/no-unused-vars)